Configure

```diff
@@ -756,11 +756,22 @@ while (@argvcopy)
 			$disabled{"shared"} = "forced";
 			$disabled{"threads"} = "forced";
 			}
-		elsif (/^-D(.*)$/)
+		elsif (/^[-\/]D(.*)$/)
 			{
 			push @user_defines, $1;
 			}
-		else	# common if (/^[-+]/), just pass down...
+		elsif (/^\/link$/)
+			{
+			# This is Visual Studio-specific thing.
+			# Consume all options that start with slash, e.g.
+			# /link /subsystem:console,5.1. Yes, it has to
+			# be last or terminated with non-slash option.
+			while (@argvcopy[0] =~ /^\//)
+				{
+				$libs.=shift(@argvcopy) . " ";
+				}
+			}
+		else	# common if (/^[-+\/]/), just pass down...
 			{
 			$_ =~ s/%([0-9a-f]{1,2})/chr(hex($1))/gei;
```

Comment:
For your arch example, I would have done this:

```
./Configure {...whatever...} '-arch armv7'
```

(that's also an argument for a more explicit (and cumulative?) `./Configure {...} --cflags='-arch armv7'`)

> But before you say that one can pass `-link%20/subsystem:console,5.1` as it is now, recall that suggestion also is to use /link to direct options to $libs(`*`), instead of $user_cflags. Yes, one can argue that one can use /link as trigger and then require `/link%20/subsystem:console,5.1`...

Gah!  No, please no, let's not add *even more* to the amount of confusion, please!

> (*) well, ideally it would be more appropriate to pass it to $user_lflags, but there is no such thing :-(

There's absolutely nothing stopping us.  Actually, I did suggest that at some point and having that go into `$(LDFLAGS)` instead of `$(EX_LIB)` (where I'd still expect all `-l` and `-L` options to end up), but that was met with scepticism...